DayrSuryan/DSEthiop3.xml

```diff
@@ -336,7 +336,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                               በረከተ፡ ቄርሎስ፡ ወኤጲፋንዮስ።  ወኵሎሙ፡ ሊቃነ።
                               ጳጳሳት፡ ወጳጳሳት። ወኤጲስ፡ ቆጶሳት<hi rend="rubric">፨</hi> ርቱዓነ፡ ሃይማኖት፡
                               <pb n="97r"/>
-                              <cb n="a"/>በረከተ፡ አባ፡ ጰን<sic resp="DR">ጦ</sic>ሌዎን፡ ወኵሎሙ፡ እለ፡ ውስተ፡ ጾማዕት<hi rend="rubric">፨</hi>
+                              <cb n="a"/>በረከተ፡ አባ፡ <sic resp="DR">ጰንጦሌዎን፡</sic> ወኵሎሙ፡ እለ፡ ውስተ፡ ጾማዕት<hi rend="rubric">፨</hi>
```

Comment:
This change is fine, but if you only made it because you want the ! at the end of the word, then you don't need to do this in the coming manuscripts, I will tell Pietro anyway.

Reply:
Ok, overkill :)